.circleci/src/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,45 @@
+when:
+  or:
+    - and:
+        - << pipeline.parameters.api_triggered >>
+        - equal: [<< pipeline.parameters.workflow_type >>, e2e_tests]
+    - and:
+        - equal: [main, << pipeline.git.branch >>]
+        - not: << pipeline.parameters.api_triggered >>
+jobs:
+  - bump_version:
+      prepare_delivery: false
+  - build_web:
+      name: build_ig_test_cms_web
+      build_config_name: integreat-test-cms
+      requires:
+        - bump_version
```

Comment:
this is actually not required for the web e2e tests and can removed here.

Reply:
It is needed as it is somehow our "setup task" perparing the environment variables.